README.md

```diff
@@ -31,33 +33,30 @@ A Model Context Protocol (MCP) server that provides browser automation capabilit
 ```
 
 
-#### Installation in VS Code
+### Usage with VS Code
 
-Install the Playwright MCP server in VS Code using one of these buttons:
+For quick installation, use one of the one-click install buttons at the top of this README.
 
-<!--
-// Generate using?:
-const config = JSON.stringify({ name: 'playwright', command: 'npx', args: ["-y", "@playwright/mcp@latest"] });
-const urlForWebsites = `vscode:mcp/install?${encodeURIComponent(config)}`;
-// Github markdown does not allow linking to `vscode:` directly, so you can use our redirect:
-const urlForGithub = `https://insiders.vscode.dev/redirect?url=${encodeURIComponent(urlForWebsites)}`;
--->
+For manual installation, add the following JSON block to your User Settings (JSON) file in VS Code. You can do this by pressing `Ctrl + Shift + P` and typing `Preferences: Open User Settings (JSON)`.
 
-[<img alt="Install in VS Code Insiders" src="https://img.shields.io/badge/VS_Code_Insiders-VS_Code_Insiders?style=flat-square&label=Install%20Server&color=24bfa5">](https://insiders.vscode.dev/redirect?url=vscode-insiders%3Amcp%2Finstall%3F%257B%2522name%2522%253A%2522playwright%2522%252C%2522command%2522%253A%2522npx%2522%252C%2522args%2522%253A%255B%2522-y%2522%252C%2522%2540playwright%252Fmcp%2540latest%2522%255D%257D)
+Optionally, you can add it to a file called `.vscode/mcp.json` in your workspace. This will allow you to share the configuration with others.
 
-Alternatively, you can install the Playwright MCP server using the VS Code CLI:
+> Note that the `mcp` key is not needed in the `.vscode/mcp.json` file.
```

Comment:
That's very unfortunate as I used to keep my config files in sync across the ecosystem. Now everyone except for VS Code is aligned?

Reply:
We can keep the original codeblock, but many readme's for MCP have instructions per editor already as the install experience differs based on files and setting UI

Reply:
I'm not sure I understand, is mcp key valid or not? If it is valid, I'd remove this note.

Reply:
@digitarald we could remove the block for `mcp.json` and just go with User Settings for simplicity.